autotag.go

```diff
@@ -203,7 +221,7 @@ func (r *GitRepo) tagNewVersion() error {
 func (r *GitRepo) parseCommit(commit *git.Commit) (*version.Version, error) {
 	var b bumper
 	msg := commit.Message()
-	log.Println("Parsing ", msg, ":")
+	log.Printf("Parsing %s: %s", commit.Id, msg)
```

Comment:
Do we want a `\n` in the format string?  (this comment might apply to multiple output messages in this repo).